dnsimple/zones_records.go

```diff
@@ -6,17 +6,18 @@ import (
 
 // ZoneRecord represents a DNS record in DNSimple.
 type ZoneRecord struct {
-	ID           int    `json:"id,omitempty"`
-	ZoneID       string `json:"zone_id,omitempty"`
-	ParentID     int    `json:"parent_id,omitempty"`
-	Type         string `json:"type,omitempty"`
-	Name         string `json:"name"`
-	Content      string `json:"content,omitempty"`
-	TTL          int    `json:"ttl,omitempty"`
-	Priority     int    `json:"priority,omitempty"`
-	SystemRecord bool   `json:"system_record,omitempty"`
-	CreatedAt    string `json:"created_at,omitempty"`
-	UpdatedAt    string `json:"updated_at,omitempty"`
+	ID           int      `json:"id,omitempty"`
+	ZoneID       string   `json:"zone_id,omitempty"`
+	ParentID     int      `json:"parent_id,omitempty"`
+	Type         string   `json:"type,omitempty"`
+	Name         string   `json:"name",omitempty`
```

Comment:
Also note the sintax is incorrect, it should be

    `json:"name,omitempty"`

not

    `json:"name",omitempty`

However, that should not be changed.